docs/slurm-plugin.md

```diff
@@ -1 +1,16 @@
 # SLURM Plugin
+
+:::{note}
+
+This has not yet been implemented.
```

Comment:
- https://github.com/transparency-certified/spank-tro  (plugin)
- https://github.com/transparency-certified/slurm-docker-cluster (Example deployment to run it)

I beg to differ ;-)